portal/src/graphql/adminapi/AuditLogScreen.tsx

```diff
@@ -37,13 +43,37 @@ function CommandBarDropdownWrapper(props: ICommandBarItemProps) {
   return <CommandBarDropdown {...dropdownProps} />;
 }
 
+function RefreshButton(props: ICommandBarItemProps) {
+  const tooltipStyle: Partial<ITooltipHostStyles> = {
+    root: { display: "inline-block" },
+  };
+  const tooltipId = useId("refreshTooltip");
+  const tooltipCalloutProps = { gapSpace: 0 };
+  const tooltipcontent =
+    "Refresh. Updated: " + formatDatetime(props.locale, getCurrentDate());
```

Comment:
Change the copy to `Last refreshed {{timestamp}}`. Note that the timestamp must be a prop which is the state. DO NOT call `new Date()` here otherwise the timestamps can be unmatched. Use luxon to format timestamp relatively.